build.sbt

```
@@ -21,7 +21,7 @@ lazy val scalatest = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "discipline-core" % "1.5.1",
       "org.scalatestplus" %%% "scalacheck-1-16" % "3.2.14.0",
```

This has to be updated as well:
```suggestion
      "org.scalatestplus" %%% "scalacheck-1-17" % "3.2.16.0",
```
Otherwise it is likely to end up with both `scalacheck-1-16` and `scalacheck-1-17` on class path.

---

This is annoying, I don't know why they still do that. scalacheck is following semver now.